reference: update TiSpark FAQ for dev and 3.0 (#1561)
pingcap/docs: dev/reference/tispark.md

Q: What can I do if `java.sql.BatchUpdateException: Data Truncated` is returned when executing SQL statements using TiSpark?

A: This error occurs because the length of the data written exceeds the length of the data type defined by the database. You can check the field length and adjust it accordingly.

> Q: What can I do if `Error：java.io.InvalidClassException: com.pingcap.tikv.region.TiRegion; local class incompatible: stream classdesc serialVersionUID ...` is returned when TiSpark executing Spark?

Reviewer: ```suggestion
Q: What can I do if `Error：java.io.InvalidClassException: com.pingcap.tikv.region.TiRegion; local class incompatible: stream classdesc serialVersionUID ...` is returned when TiSpark is executing a Spark task?
```

> A: The error message shows a `serialVersionUID` conflict, which occurs because you used `class` and `TiRegion` of different versions. Since `TiRegion` is unique to TiSpark, it is possible that there are multiple TiSpark packages. To fix this error, you need to make sure that TiSpark dependencies are the same for each node in the cluster.

Reviewer: ```suggestion
A: The error message shows a `serialVersionUID` inconsistency, which occurs because you have used `class` and `TiRegion` of different versions. Because `TiRegion` only exists in TiSpark, it might be that multiple TiSpark packages are used. To fix this error, make sure that TiSpark dependencies are consistent between the cluster and the local.
```

> A: The error message shows a `serialVersionUID` inconsistency, which occurs because you have used `class` and `TiRegion` of different versions. Because `TiRegion` only exists in TiSpark, it might be that multiple TiSpark packages are used. To fix this error, you need to make sure that TiSpark dependencies are consistent between the cluster and the local.

Reviewer 2: This line is not accurate in meaning. @TomShawn Please see [the Chinese version](https://pingcap.com/docs-cn/dev/reference/tispark/). 
@anotherrachel Please be critical when accepting suggestions.

Author: Got it. Thanks for pointing out.

Reviewer: Got it. Thanks.

> A: The error message shows a `serialVersionUID` conflict, which occurs because you have used `class` and `TiRegion` of different versions. Because `TiRegion` only exists in TiSpark, it might be that multiple TiSpark packages are used. To fix this error, you need to make sure that the versions of TiSpark dependency are consistent for each node in the cluster.

Reviewer 2: ```suggestion
A: The error message shows a `serialVersionUID` conflict, which occurs because you have used `class` and `TiRegion` of different versions. Because `TiRegion` only exists in TiSpark, multiple versions of TiSpark packages might be used. To fix this error, make sure the version of TiSpark dependency is consistent among all nodes in the cluster.
```
